config/routes.rb

```diff
@@ -8,6 +8,9 @@
   get 'feedback' => 'feedback_forms#new'
 
 
+  #define a catchall route so Rails will send the request or render the public/404.html page (in production)
+  match "*path", to: "errors#catch_404", via: :all
+
```

Comment:
Does rails match routes in the order defined? Should this be last in the route file?

Doesn't rails throw an error if nothing is found? Could this be accomplished by adding a `rescue_from` to the application controller?